src/psyclone/gocean1p0.py

```diff
@@ -719,8 +735,15 @@ def _lower_bound(self):
         All occurences of {start} and {stop} in _bounds_loopup will
         be replaced with the constant loop boundary variable, e.g.
         "{stop}+1" will become "istop+1" (or "jstop+1" depending on
-        loop type).'''
+        loop type).
 
+        :param parent:
```

Comment:
A little more verbose please for the argument descriptions :-)
And please remove the "]"

Reply:
You're so picky. As it happens, I looked a bit more closely at what `parent` is used for and realised that we can remove it altogether since it is actually always `self`.